### ISSUE TITLE
Custom Linear progress indicator

### ISSUE THREAD

**Issue by naveeramesh** [Dezenix/native-frontend-flutter#20]
### What would you like to share?

Create a custom linear progress indicator where you  can visualize the size of the file downloading respectively
Make a smooth working progress indicator by using attractive colors.
Follow the [Guidelines](https://github.com/Dezenix/native-frontend-flutter/blob/main/Guidelines.md) and code of conduct when your pushing changes.

Folder structure:
Custom widgets\custom_linearprogress01\

**Comment by msramesh2k1** [comment]
can you assign me this issue @naveeramesh

**Comment by SharviE29** [comment]
I would like to work on this issue. Would you please assign it to me? I'm a SWoC participant.

**Comment by mdnazisharman2803** [comment]
@naveeramesh , mam,kindly assign me this issue , i will work on it and please fix label thanks

**Comment by mdnazisharman2803** [comment]
Some announcements @everyone @here: 
1. All @Contributors are requested to avoid creating any new issues & work on the existing assigned ones.
2. All @Project Admins  & @Mentors  are requested to get the pending PRs on your projects resolved ASAP as the program is going to end soon.
3. We'll be circulating forms for active mentors & PAs and feedback form for contributors soon...

We'd love to hear about your experience in this 3 month long opensource journey of GSSoC'22. 

The blog section by your awesome fellow contributors/PAs/mentors is now LIVE on our website 
Check it out at: https://gssoc.girlscript.tech/blog
Do tell us if we missed giving shoutout to anyone's blog & we'll add it right away.
Feel free to write a blog on any platform & ping me with the link in#contests-at-gssoc channel

@naveeramesh  According to this message on announcement chanel.. I'm really sad to say that won't be able to work on this issue... 

Kindly unassign me mam

**Comment by naveeramesh** [comment]
But this issue wasn't created by you.

**Comment by mdnazisharman2803** [comment]
> But this issue wasn't created by you.

Yes mam but even assigned issue we can't work on so  I've unassigned..

**Comment by mdnazisharman2803** [comment]
@naveeramesh  sorry to disturb you.. Kindly assign me this issue, as some people are still working on issues even new issue, that's affecting me on leaderboard, I should probably work on this issue... 
Thank you

**Comment by mdnazisharman2803** [comment]
@naveeramesh its completed mam close this issue
